### ISSUE TITLE
Cookie value always false since 2.7.7.1

### ISSUE THREAD

**Issue by WHITE-developer** [craftcms/cms#4053]
### Description
I always get a value `false` for cookies since the craft 2.7.7.1 update

### Steps to reproduce

1. `$cookie = craft()->request->getCookie('cookie_name');`
2. `$cookie->value` => false

### Additional info

- Craft version: 2.7.7.2
- PHP version: 7.1.13
- Database driver & version: mysql 5.7.24
- Plugins & versions:

**Comment by NMathar** [comment]
I can't login with enableCsrfProtection => true 

because the value of my  CRAFT_CSRF_TOKEN is also always false but my browser has a value for this cookie.

This happens after the upgrade to 2.7.7.2

```
object(Craft\HttpCookie)#49 (9) {
  ["name"]=>
  string(16) "CRAFT_CSRF_TOKEN"
  ["value"]=>
  bool(false)
  ["domain"]=>
  string(0) ""
  ["expire"]=>
  int(0)
  ["path"]=>
  string(1) "/"
  ["secure"]=>
  bool(false)
  ["httpOnly"]=>
  bool(true)
  ["_e":"CComponent":private]=>
  NULL
  ["_m":"CComponent":private]=>
  NULL
}
```

**Comment by angrybrad** [comment]
Looking into this.

**Comment by angrybrad** [comment]
This should be fixed in 2.7.7.3.
